locksmith/__tests__/controllers/v2/ticketsController.test.ts

```diff
@@ -1,12 +1,14 @@
 import request from 'supertest'
 import { loginRandomUser } from '../../test-helpers/utils'
+const metadataOperations = require('../../../src/operations/metadataOperations')
 
 const app = require('../../../src/app')
 
 jest.setTimeout(600000)
 const lockAddress = '0x3F09aD349a693bB62a162ff2ff3e097bD1cE9a8C'
+const wrongLockAddress = '0x00'
 const network = 4
-const tokenId = '123'
+const tokenId = '2244'
```

Comment:
Why the change?

Reply:
now the endpoint currently avoids overrides, the old stored value for this tokenId have a not valid structure